elastic/kibana
# [Fleet] Create enrollment API keys as current user

## Summary

As part of the move to fleet server we will remove the current `fleet_enroll` user. That user was used to create API keys.
In that PR we change that to use the current user to create API keys instead.

I also removed useless argument to the invalidate calls

Also I added metatada to the enrollment API key.

## How to test 

This PR should not change anything.
* the fleet agent setup  should create API keys
* you should be able to create and invalidate API keys.

## Automated test

this part of the code is covered by the fleet api integration tests
